pkg/scaffold/crd.go

```diff
@@ -145,25 +151,30 @@ func (s *CRD) CustomRender() ([]byte, error) {
 	return k8sutil.GetObjectBytes(dstCRD)
 }
 
-func newCRDForResource(r *Resource) *apiextv1beta1.CustomResourceDefinition {
-	return &apiextv1beta1.CustomResourceDefinition{
+func (s *CRD) newCRD() (*apiextv1beta1.CustomResourceDefinition, error) {
```

Comment:
No need to return an error.

Reply:
right, this is a left-over form a previous idea, and I actually don't need it anymore

Reply:
fixed in 543b7d0